quodlibet/ext/events/synchronizedlyrics.py

```diff
@@ -28,21 +28,23 @@ class SynchronizedLyrics(EventPlugin, PluginConfigMixin):
     PLUGIN_ID = 'SynchronizedLyrics'
     PLUGIN_NAME = _('Synchronized Lyrics')
     PLUGIN_DESC = _('Shows synchronized lyrics from an .lrc file '
-                    'with same name as the track (or similar).')
+                    'with same name as the track (or similar), or '
+                    'embedded tags.')
     PLUGIN_ICON = Icons.FORMAT_JUSTIFY_FILL
 
     SYNC_PERIOD = 10000
 
     DEFAULT_BGCOLOR = '#343428282C2C'
-    DEFAULT_TXTCOLOR = '#FFFFFFFFFFFF'
+    DEFAULT_HLCOLOR = '#343428282C2C'
+    DEFAULT_TXTCOLOR = '#FFFFFFFFFFF'
     DEFAULT_FONTSIZE = 25
 
     CFG_BGCOLOR_KEY = "backgroundColor"
+    CFG_HLCOLOR_KEY = "highlightColor"
     CFG_TXTCOLOR_KEY = "textColor"
     CFG_FONTSIZE_KEY = "fontSize"
 
-    # Note the trimming of whitespace, seems "most correct" behaviour
-    LINE_REGEX = re.compile(r"\s*\[([0-9]+:[0-9.]*)\]\s*(.+)\s*")
+    LINE_REGEX = re.compile(r"<[\d]{2}:[\d]{2}\.[\d]{2}>|\[[\d]{2}:[\d]{2}\.[\d]{2}\]")
```

Comment:
Because the regex would get confused if the line happens to contain either one of these <> []. This regex only matches the timestamps, and then the `_define_groups` function takes care of assigning the text to each timestamp. That said, I'm not the best at regex, feel free to make improvements.